test/e2e/kube-ovn/ipam/ipam.go

```diff
@@ -509,4 +509,56 @@ var _ = framework.Describe("[group:ipam]", func() {
 		deploy = deployClient.PatchSync(deploy, patchedDeploy)
 		checkFn()
 	})
+
+	framework.ConformanceIt("should allocate right IPs for the statefulset when there are multiple IP Pools added to its namespace", func() {
+		replicas := 1
+		ipsCount := 12
+		ipsRange1 := framework.RandomIPPool(cidr, ipsCount)
+		ipsRange2 := framework.RandomIPPool(cidr, ipsCount)
```

Comment:
```suggestion
		ipsRange2 := framework.RandomIPPool(testCidr, ipsCount)
```

Should be `testCidr` here?